source/iostreams.tex

```diff
@@ -1509,7 +1509,7 @@
 \tcode{parray[idx]}.
 On failure a valid
 \tcode{void*\&}
-initialized to 0.
+initialized to the null pointer value.
```

Comment:
This doesn't make any sense, with or without your change. You can't initialize a `void*&` to a null pointer value. To what object is this reference supposed to be bound? This seems like we need an LWG issue to say what this actually does.